examples/devsite-landing-demo/toolbox.js

```diff
@@ -649,71 +649,6 @@ let toolboxJson = {
       ],
     },
 
-    {
-      // Colour Category
-      kind: 'CATEGORY',
```

Comment:
We don't save any of the state of the devsite landing demo right?

Reply:
Correct